plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerTypeMapping.java

```diff
@@ -316,6 +322,11 @@ public void testDate()
 
         SqlDataTypeTest testsTrino = SqlDataTypeTest.create()
                 .addRoundTrip("date", "NULL", DATE, "CAST(NULL AS DATE)")
+                // first day of AD
+                .addRoundTrip("date", "DATE '1-01-01'", DATE, "DATE '1-01-01'")
```

Comment:
why not `0001-01-01`?

Reply:
changed to `0001-01-01` to be aligned with above cases